### PR TITLE
FLUSH commands and functions

### PR DESCRIPTION
**Pull request by LiorKogan**
Added a note: functions are not deleted.

### CODE REVIEW

**Comment by oranagra** [comment]
@LiorKogan do you expect me to read the diff and somehow figure out what was changed in that long line and why??
please add a description to the PR to make it easier.

**Comment by LiorKogan** [comment]
Northing was changed in that long line. I change "note" to "notes" and added a new note - functions are not deleted.

**Review by oranagra — APPROVED** [review]

File: content/commands/flushall/index.md [hunk 1]
```diff
@@ -62,8 +62,11 @@ It is possible to use one of the following modifiers to dictate the flushing mod
 * `ASYNC`: flushes the databases asynchronously
 * `SYNC`: flushes the databases synchronously
 
-Note: an asynchronous `FLUSHALL` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
+## Notes
+
+* An asynchronous `FLUSHALL` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
+* This commands does not delete functions
```
Comment (dwdougherty) [inline content/commands/flushall/index.md:68]:
```suggestion
* This command does not delete functions.
```

File: content/commands/flushdb/index.md [hunk 2]
```diff
@@ -62,8 +62,11 @@ It is possible to use one of the following modifiers to dictate the flushing mod
 * `ASYNC`: flushes the database asynchronously
 * `SYNC`: flushes the database synchronously
 
-Note: an asynchronous `FLUSHDB` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
+## Notes
+
+* An asynchronous `FLUSHDB` command only deletes keys that were present at the time the command was invoked. Keys created during an asynchronous flush will be unaffected.
+* This commands does not delete functions
```
Comment (dwdougherty) [inline content/commands/flushdb/index.md:68]:
```suggestion
* This command does not delete functions.
```

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Just a couple of minor issues.

**Review by dwdougherty — APPROVED** [review]

**Review by dwdougherty — APPROVED** [review]
LGTM. Merging.

**Pull request merged**